packages/react/src/components/header/components/headerLanguageSelector/HeaderLanguageSelector.module.scss

```diff
@@ -53,16 +53,14 @@ button.item,
 [type='button'].item {
   appearance: none;
   background: transparent;
-  border: none;
   color: var(--lang-selector-item-font-color);
   cursor: pointer;
   line-height: var(--lineheight-l);
+  outline: 0;
   white-space: nowrap;
 
   &:focus-visible {
-    border: none;
-    outline: var(--header-focus-outline-width) solid var(--header-focus-outline-color);
-    outline-offset: var(--header-focus-outline-width);
+    border: 3px solid var(--header-focus-outline-color);
```

Comment:
Hol' up, you can use `var(--header-focus-outline-width)` instead of 3px here.

Reply:
Ah, yes. Changed now. And also on line 48.